content/en/blog/_posts/2021-09-07-data-duplication-in-data-heavy-k8s-env.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: blog
+title: "How to Handle Data Duplication in Data-Heavy Kubernetes Environments"
+date: 2021-09-07
+slug: how-to-handle-data-duplication-in-data-heavy-kubernetes-environments 
+---
+
+**Authors:**
+Augustinas Stirbis (CAST AI)
+
+# How to Handle Data Duplication in Data-Heavy Kubernetes Environments
+
+It’s convenient to create a copy of your application with a copy of its state for each team. 
+For example, you might want a separate database copy to test some significant schema changes 
+or develop other disruptive operations like bulk insert/delete/update...
+
+**Duplicating data takes a lot of time.** That’s because you need first to download 
+all the data from a source like object storage (S3 or Cloud Storage) and then send 
+it back to a storage back-end like EBS or Standard Persistent Disk. There’s a lot of network traffic and CPU/RAM used in this process.
+
+Hardware acceleration by offloading certain expensive operations is **always a huge 
+performance boost**. It reduces the time required to complete an operation by orders 
+of magnitude - for example, GPU computation while you’re watching a high-resolution 
+video on Youtube, mining crypto, or training an ML model, CPU instructions for AES 
+disk encryption or TLS, TCP offload (TOE), and many many more.
```

Comment:
I think this paragraph is unnecessary. The previous two sufficiently lay out the problem. I appreciate the problem/solution framing.

Reply:
I cut it in half and amended to previous paragrph